sonarqube/README.md

```diff
@@ -207,6 +207,54 @@ Collecting logs is disabled by default in the Datadog Agent. To enable it, see [
 <!-- xxz tab xxx -->
 <!-- xxz tabs xxx -->
 
+#### Components Discovery
+
+You can configure how your components are discovered with the `components_discovery` parameter.
+
+`limit`
+: Maximum number of items to be auto discovered.  
+**Default value**: `10`
+
+`include`
+: Mapping of regular expressions keys and component config values that will be auto discovered.  
+**Default value**: empty map
+
+`exclude`
+: List of regular expressions with the patterns of components that will not be auto discovered.  
+**Default value**: empty list
+
+Examples:
+
+Include a maximum of `5` components with names starting with `my_project`:
+
+```yaml
+components_discovery:
+  limit: 5
+  include:
+    'my_project*':
+```
+
+Include a maximum of `20` components and exclude those beginning with `temp`:
+
+```yaml
+components_discovery:
+  limit: 20
+  include:
+    '*':
+  exclude:
+    - 'temp*'
+```
+
+Include all components with names starting with `issues` processing them with `issues_project`, and only collect metrics belonging to the category `issues`. As `limit` is not defined, the number of components discovered is limited to the default value `10`:
```

Comment:
Apologies if my [initial suggestion](https://github.com/DataDog/integrations-core/pull/13450#discussion_r1039832655) would have introduced inaccuracy here -- is the following text meant to indicate that only components with the `issues_project` tag are included?
> processing them with `issues_project`

Or does using the `tag` key do something else, such as apply the tag?

Reply:
Yes, the `tag` key indicates which tag will be applied